### ISSUE TITLE
macOS 10.13: Finder issues for non-admin users

### ISSUE THREAD

**Issue by mazimmer** [vgough/encfs#454]
On macOS 10.13 High Sierra, using osxfuse 3.7.1.

As user that is member of group admin: Using a mounted encfs folder works as expected - "fuse drive" is shown in sidebar, content of folder is shown,...

As non-admin user: Instead of "fuse drive", a plain document icon is shown that remains even when unmounting the encfs folder; content of encfs folder is always empty. 
From shell, everything is normal.
<img width="722" alt="screen shot 2017-11-20 at 12 06 02" src="https://user-images.githubusercontent.com/4983700/33015489-44389b36-cdeb-11e7-89dc-c4aa8523e983.png">

**Comment by benrubson** [comment]
Same as #440 ?

According to #341, you could have to mount as `root` with `-o allow_other`.
Please test and let us know 👍

**Comment by mazimmer** [comment]
Yes, it works with root, but the issue only occurred with High Sierra; 10.12 and below are fine for none-root users.

Not the same as #440 IMHO, since this occurs on local mount points, no smb involved, and only for 10.13.

And I just noticed, when mounting again, that for once the "fuse folder" and content appeared, but were gone after unmount/remount.
